### PR TITLE
[Symfony] Adding new packages

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php [hunk 1]
```diff
@@ -9,6 +9,14 @@ class AwsPackagesProvider
     public static function getAllServices(): array
     {
         return [
+            'cloudformation' => [
```
Comment (Nyholm) [inline src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: - cloudFormation?
- cloud_formation?

Comment (jderusse) [reply src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: `cloud-formation` ?
or `CloudFormation` like the official bundle https://github.com/aws/aws-sdk-php-symfony

Comment (Nyholm) [reply src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: This goes into the yaml config. That is why Im asking. Having dash in yaml isnt really nice, is it?

Comment (Nyholm) [reply src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: We should use snake case to be consistent with other config: 
https://symfony.com/doc/current/reference/configuration/framework.html

Comment (jderusse) [reply src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: don't Symfony use snakeCase to separate words? like `max_redirects` but use all lowercase for the same word? like `passphrase`?
just asking.

Comment (Nyholm) [reply src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php]: Yes. Symfony use snake_case to separate words. I think `Cloud Formation` is two words.

**Pull request merged**